pkg/bpf/exporter.go

```diff
@@ -111,7 +114,7 @@ func (e *exporter) attach() error {
 		return fmt.Errorf("error attaching sched_switch tracepoint: %v", err)
 	}
 
-	if config.ExposeIRQCounterMetrics() {
+	if config.IsExposeIRQCounterMetrics() {
```

Comment:
My preference is to leave it as is .. 
`IsExposeIRQCounterMetrics` does not seem right. If we have to use `Is` for bool functions, I think the better option is `IsIRQCounterMetricsExposed` just like an English sentence.